uisdk/src/main/res/drawable/kh_uisdk_ic_electric.xml

```diff
@@ -0,0 +1,13 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="16dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="M13,2l-10,12l9,0l-1,8l10,-12l-9,0l1,-8z"
+      android:strokeLineJoin="round"
+      android:strokeWidth="2"
+      android:fillColor="#00000000"
+      android:strokeColor="@color/black"
```

Comment:
perhaps export this color and call it something like `kh_uisdk_tags_icon` or `kh_uisdk_icon` ? (same in other icon assets )

Reply:
Done

Reply:
<color name="kh_uisdk_tag_icon_fill_color">#00000000</color>
 <color name="kh_uisdk_tag_icon_stroke_color">#000</color>